bulk_indexer.go

```diff
@@ -218,11 +233,21 @@ func (b *BulkIndexer) Len() int {
 	return b.buf.Len()
 }
 
+// UncompressedLen returns the number of uncompressed buffered bytes.
+func (b *BulkIndexer) UncompressedLen() int {
+	return b.writer.count
+}
+
 // BytesFlushed returns the number of bytes flushed by the bulk indexer.
 func (b *BulkIndexer) BytesFlushed() int {
 	return b.bytesFlushed
 }
 
+// BytesUncompFlushed returns the number of uncompressed bytes flushed by the bulk indexer.
+func (b *BulkIndexer) BytesUncompFlushed() int {
```

Comment:
nit: I'm fine with using longer names where it helps readability and I think would be good for consistency to use `BytesUncompressedFlushed()` here.

Reply:
I agree! updated.